app/src/main/java/com/example/androidacademymsk/FragmentMoviesDetails.kt

```diff
@@ -6,9 +6,13 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 
 class FragmentMoviesDetails : Fragment() {
 
+    private var castRecycler: RecyclerView? = null
```

Comment:
I guess `lateinit var` is better than nullable property

Reply:
Done